src/barrier-picker/SpreadBarrierCard.js

```diff
@@ -42,7 +42,7 @@ export default class SpreadBarrierCard extends Component {
                 <InputGroup
                     type="number"
                     label={`Amount per point (${currency})`}
-                    value={spreadInfo.amountPerPoint}
+                    defaultValue={spreadInfo.amountPerPoint}
```

Comment:
Actions are used to change the value and from the same input group whose value has been fixed? Or do we have a different UI for that  ?